specification/sql/resource-manager/readme.md

```diff
@@ -305,6 +305,7 @@ input-file:
 - Microsoft.Sql/stable/2014-04-01/firewallRules.json
 - Microsoft.Sql/stable/2014-04-01/geoBackupPolicies.json
 - Microsoft.Sql/stable/2014-04-01/metrics.json
+- Microsoft.Sql/stable/2014-04-01-legacy/replicationLinks_legacy.json
```

Comment:
please add - ./Microsoft.Sql/preview/2021-02-01-preview/ReplicationLinks.json in to the bottom of this section.

Reply:
I accidentally added `replicationLinks_legacy` to `package-composite-v4` tag instead of `v5` , that's why `ReplicationLinks.json` was missing. 
fixed